src/compiler/compile/render_ssr/handlers/Head.ts

```diff
@@ -12,5 +12,5 @@ export default function(node: Head, renderer: Renderer, options: RenderOptions)
 	renderer.render(node.children, head_options);
 	const result = renderer.pop();
 
-	renderer.add_expression(x`$$result.head += ${result}, ""`);
+	renderer.add_expression(x`$$result.head += '<!-- HEAD_START data-svelte="${node.id}" -->' + ${result} + '<!-- HEAD_END data-svelte="${node.id}" -->', ""`);
```

Comment:
the reason of having `data-svelte` is because it was added as an attribute.

since this is in the comments, we could have it simpler, eg: `HEAD_START_${node.id}`

or we could have it easier for us to do the querying later, eg:

`HEAD_${node.id}_START`

then we can claim it by checking if it starts with `HEAD_${node.id}` and if so, check whether it is to `_START` or to `_END`

Reply:
Good call. I honored the `data-svelte` just in case.